my-app/.eslintrc

```
@@ -0,0 +1,36 @@
+{
+  "extends": [
+    "react-app",
+    "plugin:prettier/recommended"
+  ],
+  "plugins": ["prettier"],
+  "rules": {
+//    "no-console": "error",

```

лучше поставить `warn` а не просто закомменчивать, консоль логи нужно не забывать убирать
 ```suggestion
    "no-console": "warn",
```